scripts/.phantom-run-tests.js

```diff
@@ -0,0 +1,41 @@
+var page = require('webpage').create();
```

Comment:
Does this commit [1] convince you it is a different execution environment?
[1] daba905